### PR TITLE
x11-wm/lumina: initial ebuild.

### PR DESCRIPTION
**Pull request by rubin55**
Package-Manager: portage-2.2.26

### CODE REVIEW

**Comment by rubin55** [comment]
https://bugs.gentoo.org/show_bug.cgi?id=578114

**Comment by idella** [comment]
@gentoo/proxy-maint

File: x11-wm/lumina/lumina-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+RDEPEND="${DEPEND}
+	x11-misc/numlockx
+	x11-wm/fluxbox
+	x11-apps/xbacklight
+	media-sound/alsa-utils
+	sys-power/acpi
+	app-admin/sysstat"
+
+src_compile(){
+	eqmake5 PREFIX="${ROOT}usr" L_ETCDIR="${ROOT}etc" LIBPREFIX="${ROOT}usr/$(get_libdir)" DESTDIR="${D}" CONFIG+="NO_I18N"
```
Comment (kensington) [inline x11-wm/lumina/lumina-9999.ebuild]: eqmake usually would go in ``src_configure``.

**Comment by blueness** [comment]
There are a lot of commits here.  Can you please squash them into one commit which adds a working version of the ebuild.

**Comment by monsieurp** [comment]
@blueness this is a WIP.

**Comment by blueness** [comment]
@monsieurp edumacateme what's a WIP?

**Comment by monsieurp** [comment]
a work in progress :)

**Comment by blueness** [comment]
@monsieurp no problem, but still we need to squash it.  i can do it at my end if i wind up committing it.

**Comment by idella** [comment]
It is just as well if you 'squash' it as requested. The goal is for user to prepare the patch in a final form.

**Comment by blueness** [comment]
this fails to build.  can we please get a fixed ebuild that builds else i'll mask it.

**Pull request closed without merging**